contracts/LaunchpegFactory.sol

```diff
@@ -24,14 +24,15 @@ contract LaunchpegFactory is
     address public override flatLaunchpegImplementation;
 
     /// @notice Default fee percentage
+    /// @dev In basis points e.g 100 for 1%
     uint256 public override joeFeePercent;
     /// @notice Default fee collector
     address public override joeFeeCollector;
 
-    /// @notice Checks if an address is stored as a Launchpeg
-    mapping(address => bool) public override isLaunchpeg;
-    /// @notice Launchpegs address list
-    address[] public override allLaunchpegs;
+    /// @notice Checks if an address is stored as a Launchpeg, by type of Launchpeg
+    mapping(uint8 => mapping(address => bool)) public override isLaunchpeg;
```

Comment:
pls no uint8, it's worse than uint256 as it adds a cast and is still stored in 256 bits.
Smaller types (smaller than 256 bits) are useful **only** in structure where your aim is to pack the most variables into multiples of 256 bits (because there it's so much better)